tests/e2e/tensor_ops/CMakeLists.txt

```diff
@@ -45,11 +45,28 @@ iree_check_single_backend_test_suite(
     check_vmvx_local-task
   SRCS
     "extract_slice.mlir"
+    "pack.mlir"
     "tensor_insert_slice.mlir"
+    "unpack.mlir"
+  TARGET_BACKEND
+    "vmvx"
+  DRIVER
+    "local-task"
+)
+
+iree_check_single_backend_test_suite(
+  NAME
+    check_vmvx_ukernel_local-task
+  SRCS
+    "pack.mlir"
+    "unpack.mlir"
   TARGET_BACKEND
     "vmvx"
   DRIVER
     "local-task"
+  COMPILER_FLAGS
+    "--iree-vmvx-enable-microkernels"
+    "--iree-vmvx-enable-microkernels-decompose-linalg-generic=false"
 )
```

Comment:
This test is failing on Windows

https://github.com/iree-org/iree/actions/runs/4167636144/jobs/7213463962
```
[ RUN      ] module.static_unpack_simple
C:\a\iree\iree\runtime\src\iree\modules\check\module.cc(269): error: Failed
Expected equality of these values. Contents does not match.
  lhs:
    4x4xi32=[0 0 0 0][0 0 0 0][0 0 0 0][0 0 0 0]
  rhs:
    4x4xi32=[0 1 2 3][4 5 6 7][8 9 10 11][12 13 14 15]
```
(and other errors)

Looks similar to the other ukernel failures on Windows. https://github.com/iree-org/iree/blob/b02dfa948b7ba7b28b34b5f06d0ff00a03be303c/build_tools/cmake/ctest_all.sh#L68-L92

Reply:
> This test is failing on Windows
> 
> https://github.com/iree-org/iree/actions/runs/4167636144/jobs/7213463962
> 
> ```
> [ RUN      ] module.static_unpack_simple
> C:\a\iree\iree\runtime\src\iree\modules\check\module.cc(269): error: Failed
> Expected equality of these values. Contents does not match.
>   lhs:
>     4x4xi32=[0 0 0 0][0 0 0 0][0 0 0 0][0 0 0 0]
>   rhs:
>     4x4xi32=[0 1 2 3][4 5 6 7][8 9 10 11][12 13 14 15]
> ```
> 
> (and other errors)
> 
> Looks similar to the other ukernel failures on Windows.
> 
> https://github.com/iree-org/iree/blob/b02dfa948b7ba7b28b34b5f06d0ff00a03be303c/build_tools/cmake/ctest_all.sh#L68-L92

Thanks for reporting it.. Yes, the tensor.pack tests go with the same pipeline as existing unpack ops. Let me create a PR to disable the tests on windows.